schedule/yast/opensuse/yast2_ncurses/yast2_ncurses.yaml

```diff
@@ -25,10 +25,18 @@ schedule:
   - console/yast2_lan_hostname/dhcp_no
   - console/yast2_lan_hostname/dhcp_yes
   - console/yast2_lan_hostname/dhcp_yes_eth
-  - console/yast2_dns_server
+  - console/yast2_dns_server_wizard_service_active_enabled
+  - console/yast2_dns_server_service_inactive_enabled
+  - console/yast2_dns_server_service_active_disabled
+  - console/yast2_dns_server_service_inactive_disabled
+  - console/yast2_dns_server_service_abort_keep
   - console/yast2_nfs_client
   - console/yast2_snapper_ncurses
 test_data:
   yast2_lan_hostname:
     ui: ncurses
     confirm_warning: 1
+  install_packages:
```

Comment:
we don't need this as it is set in the parent job:
https://openqa.opensuse.org/tests/2918924#

Reply:
updated